core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -211,11 +213,16 @@ private String toText(boolean verbose, int level)
         return new TextRenderer(verbose, level).render(representation);
     }
 
-    private String toJson()
+    private String toLogicalJsonPlan()
     {
         return new JsonRenderer().render(representation);
     }
 
+    public Map<Integer, List<DistributedPlanRepresentation>> toDistributedPlanRepresentation(boolean verbose)
```

Comment:
This method doesn't belong in this class -- it's not about rendering but about converting to DistributedPlanRepresentation.

Also, it's weird that it would rely on the JsonRenderer to do so.

The abstraction should really be:

Distributed Plan -> Distributed Plan Representation -> Textual form of the rendered plan